src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    if (Animal.alive === undefined) {
+      Animal.alive = [];
+    }
+
+    if (health > 0) {
```

Comment:
then if we setted up always health animal by 100 we always want to push this animal. Remove this if and move `Animal.alive.push(this);` to the constructor right after `this.health = 100;`

```
this.health = health;
Animal.alive.push(this);
```